composer.json

```diff
@@ -10,7 +10,7 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "illuminate/support": "~5.0"
+        "illuminate/support": "~6.0"
```

Comment:
Ideally you could use         `"illuminate/support": "^5.0|^6.0"` to maintain BC and continue supporting laravel 5